code/datums/storage/subtypes/internal.dm

```diff
@@ -0,0 +1,467 @@
+
+/datum/storage/internal/vest/New(atom/parent)
+	. = ..()
+	set_holdable(can_hold_list = list(
+		/obj/item/stack/razorwire,
+		/obj/item/stack/sheet,
+		/obj/item/stack/sandbags,
+		/obj/item/stack/snow,
+	))
+
+/datum/storage/internal/white_vest
+	max_w_class = WEIGHT_CLASS_BULKY
+	storage_slots = 6 //one more than the brown webbing but you lose out on being able to hold non-medic stuff
+	max_storage_space = 24
+
+/datum/storage/internal/white_vest/New(atom/parent)
+	. = ..()
+	set_holdable(can_hold_list = list(
+		/obj/item/healthanalyzer,
+		/obj/item/stack/medical,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/reagent_containers/hypospray/advanced,
+		/obj/item/reagent_containers/hypospray/autoinjector,
+		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/reagent_containers/pill,
+		/obj/item/storage/pill_bottle,
+		/obj/item/clothing/glasses/hud/health,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/tweezers,
+		/obj/item/tweezers_advanced,
+		/obj/item/bodybag,
+		/obj/item/roller,
+		/obj/item/whistle,
+	))
+
+/datum/storage/internal/surgery_webbing
+	storage_slots = 12
+	max_storage_space = 24
+
+/datum/storage/internal/surgery_webbing/New(atom/parent)
+	. = ..()
+	set_holdable(can_hold_list = list(
+		/obj/item/tool/surgery,
+		/obj/item/stack/nanopaste,
+		/obj/item/tweezers,
+		/obj/item/tweezers_advanced,
+	))
+
+/datum/storage/internal/holster
+	storage_slots = 4
+	max_storage_space = 10
+	max_w_class = WEIGHT_CLASS_BULKY
+
+/datum/storage/internal/holster/New(atom/parent)
+	. = ..()
+	set_holdable(can_hold_list = list(
+		/obj/item/weapon/gun/pistol,
+		/obj/item/ammo_magazine/pistol,
+		/obj/item/weapon/gun/revolver,
+		/obj/item/ammo_magazine/revolver,
+		/obj/item/weapon/gun/energy/lasgun/lasrifle/standard_marine_pistol,
+		/obj/item/cell/lasgun/lasrifle,
+	))
+	storage_type_limits_max = list(/obj/item/weapon/gun = 1)
+
+/datum/storage/internal/modular
+	max_storage_space = 2
+	storage_slots = 2
+	max_w_class = WEIGHT_CLASS_TINY
+
+/datum/storage/internal/modular/New(atom/parent)
+	. = ..()
+	set_holdable(
+		can_hold_list = list(/obj/item/stack),
+		storage_type_limits_list = list(/obj/item/clothing/glasses)
+	)
+
+/datum/storage/internal/pocket
+	max_storage_space = 6
+	storage_slots = 2
+	max_w_class = WEIGHT_CLASS_NORMAL
+
+/datum/storage/internal/pocket/New(atom/parent)
+	. = ..()
+	set_holdable(
+		cant_hold_list = list(/obj/item/cell/lasgun/volkite/powerpack),
+		storage_type_limits_list = list(
+			/obj/item/ammo_magazine/rifle,
+			/obj/item/cell/lasgun,
+			/obj/item/ammo_magazine/smg,
+			/obj/item/ammo_magazine/pistol,
+			/obj/item/ammo_magazine/revolver,
+			/obj/item/ammo_magazine/sniper,
+			/obj/item/ammo_magazine/handful,
+		)
+	)
+
+/datum/storage/internal/pocket/insertion_message(obj/item/item, mob/user)
+	var/visidist = item.w_class >= WEIGHT_CLASS_NORMAL ? 3 : 1
+	//Grab the name of the object this pocket belongs to
```

Comment:
what is this var name

Reply:
I  have no idea lmao, this is some proper copypasta